dotman-plugin/src/main/java/net/minevn/dotman/database/LogDAO.kt

```diff
@@ -194,6 +196,29 @@ abstract class LogDAO : DataAccess() {
         }
     }
 
+    fun getTransactionDetailsById(id: String, sender: CommandSender) = run {
+        getTransactionDetailsById().statement {
+            setString(1, id)
+            fetchRecords {
+                val name = getString("name")
+                val type = getString("type")
+                val price = getInt("price")
+                val time = getLong("time").timeToString()
+                val pointsnhan = getInt("pointsnhan")
+                getMain().language.transactionIdDetailsOutPut.forEach() {
+                    it  .replace("%TRANSACTION_ID%", id)
+                        .replace("%PLAYER%", name)
+                        .replace("%CARD_TYPE%", type)
+                        .replace("%CARD_PRICE%", price.format())
+                        .replace("%POINTS_RECEIVED%", pointsnhan.toString())
+                        .replace("%POINT_UNIT%", getMain().config.pointUnit)
+                        .replace("%DATE%", time)
+                        .apply { sender.sendMessage(this) }
```

Comment:
DAO class chỉ có nghĩa vụ trả về giá trị, còn công việc này nên đưa ra ngoài

Reply:
> DAO class chỉ có nghĩa vụ trả về giá trị, còn công việc này nên đưa ra ngoài

@coderabbitai learn this rule

Reply:
> DAO class chỉ có nghĩa vụ trả về giá trị, còn công việc này nên đưa ra ngoài

em đã đưa send message ra ngoài